release-notes/v2.2.0.md

```diff
@@ -21,7 +21,7 @@ The following table lists the resolved issues for HPE Alletra 9000 and Primera/3
 |ID|Component |Title|Resolution|
 |--|---------|-----|-----------|
 |CON-2642|csi.k8s|Duplicate LUN Creation : When a PVC is attached to a pod, occasionally the CSI driver creates duplicate LUN's for the same PVC,Host,port combination| If the array is busy, and rejects LUN creation requests from CSP, a retry did not track whether the LUN is created, but triggered another LUN creation call (hence duplication). Now, we query what LUN's are already created and only retry the subset.|
-|CON-2166|csi.k8s|Cloning from Primera or 3PAR Snapshot using importVolAsClone feature fails.|The CSP has a process for clones with PVC's as a source; it first creates a intermediate snapshot, creates clone with this intermediate snapshot as the source, and then deletes the snapshot. This process has a side effect of deleting the snapshot always (a bug, because CSP does not check whether the snapshot was user created or CSI generated intermediate), and the fix adds the necessary check.|
+|CON-2166|csi.k8s|Cloning from Primera or 3PAR Snapshot using importVolAsClone feature fails.|The CSP has a process for clones with PVC's as a source; it first creates a intermediate snapshot, creates clone with this intermediate snapshot as the source, and then deletes the snapshot. This process has a side effect of deleting the snapshot always (a bug, because CSP does not check whether the snapshot was user created ones or CSI generated intermediate), and the fix adds the necessary check.|
```

Comment:
Can you please rephrase the title as per the COn-2166 
Unable to clone from Primera or 3PAR Snapshot